src/entities/requests/UpdateUserRequest.ts

```diff
@@ -0,0 +1,5 @@
+export default class UpdateUserRequest {
+  username?: string;
```

Comment:
The validation is handled by tsoa, which also generates the swaggerfile. The username is optional, because otherwise, you would need to introduce multiple request-entities (e.g. the user wants to only update username and mail, or username alone)